mws/resource_mws_vpc_endpoint.go

```diff
@@ -76,6 +76,13 @@ func ResourceMwsVpcEndpoint() *schema.Resource {
 	s := common.StructToSchema(VPCEndpoint{}, func(s map[string]*schema.Schema) map[string]*schema.Schema {
 		// nolint
 		s["vpc_endpoint_name"].ValidateFunc = validation.StringLenBetween(4, 256)
```

Comment:
For line 78 s["vpc_endpoint_name"].ValidateFunc = validation.StringLenBetween(4, 256):
1. AWS and GCP have different restriction on the string length.
2. We have validation check for the input string length for AWS and GCP in the backend. A user error will be reported by the server.

In this case, do we need to add specific override for GCP here? @nkvuong

Reply:
Terraform does not support more complex`ValidateFunc`, so for now we should just set the string length validation to cover both AWS & GCP